packages/core/src/client/decorators/bundlerClient.ts

```diff
@@ -88,7 +88,7 @@ export type BundlerActions = {
   getUserOperationReceipt(hash: Hash): Promise<UserOperationReceipt | null>;
 
   /**
-   * calls `eth_supportedEntryPoints` and returns the entrypoints the RPC
+   * calls `eth_supportedEntryPoints` and returns the entryPoints the RPC
```

Comment:
This is such a nitpick I feel bad about writing it, when appearing in prose not referring to a specific variable, it could potentially be written as "entry points." Likewise throughout.

Reply:
naw good shout, lemme address this